connectors.md

```diff
@@ -42,15 +106,24 @@ Step 4: Adding models to the context
 app.use('/graphql', apolloServer({
   schema: Schema,
   models: {
-    new Author({ connector: mongo }),
-    new Post({ connector: mongo }),
+    Author: new Author({ connector: mongo }),
+    Post: new Post({ connector: mongo }),
   }
 });
 ```
 
-Step 4: Calling connectors in resolve functions
+Step 4: Calling models in resolve functions
 ```
-function resolve(parent, args, ctx, info){
-  return ctx.models.Post.get(args.id);
+function resolve(author, args, ctx){
+  return ctx.models.Post.getByAuthor(author.id);
 }
 ```
+
+## FAQ
+
+
+**Question:** Are models the same as GraphQL types?
+**Answer:** There will usually be a 1:1 correspondence between types in your schema and the models, so it makes sense to keep them in the same file, or at least in the same folder. The subtle difference is that while the GraphQL schema describes the types and their relationships, the models define which connectors should be used to fetch the actual data for that type.
+
+**Question:** Can I use <Mongoose/Sequelize/MyOtherFavoriteORM> with Apollo Server?
+**Answer:** Yes, you can use an existing ORM if you wish. In that case, the models you use will be your ORM's models, and the connector part will be in the ORM itself. Most ORMs don't implement batching and caching, so you may need to do that manually where it becomes necessary, or move some of your models to a connector that automatically does batching and caching.
```

Comment:
I'd say the long-term plan is probably make a Mongo connector that can be injected into Mongoose, for example. This paragraph makes it sound a bit harder than it is IMO.